scripts/tests

```diff
@@ -20,9 +23,12 @@ def serial_ports():
     if platform.system() == "Windows":
         ports = ["COM%s" % (i + 1) for i in range(256)]
     elif platform.system() == "Darwin":
-        ports = glob.glob("/dev/tty.usbserial*")
+        ports = glob.glob("/dev/tty.usb*")
+
+        r = re.compile(".*tty.usb(serial-|modem)[0-9]+$")
+        ports = [p for p in ports if r.match(p)]
     elif platform.system() == "Linux":
-        ports = glob.glob("/dev/ttyUSB*")
+        ports = glob.glob("/dev/ttyUSB*") + glob.glob("/dev/ttyACM*")
     else:
```

Comment:
Please add an option to manually pass the serial device here. We call this `-u` /`--uart` in pqm4.